shell/platform/darwin/ios/framework/Source/FlutterUIPressProxy.mm

```diff
@@ -2,9 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterUIPressProxy.h"
+
+#import "flutter/shell/platform/darwin/common/framework/Headers/FlutterMacros.h"
+
 #import <UIKit/UIKit.h>
 
-#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterUIPressProxy.h"
+FLUTTER_ASSERT_ARC
 
 @interface FlutterUIPressProxy ()
 @property(nonatomic, readonly) UIPress* press;
```

Comment:
Were these properties accidentally weak before and it just hadn't happened to break anything?

Reply:
Interesting - yeah seems like this should previously have been explicitly `strong`, and maybe worked because (I assume) it's retained in that map we create to pointer events being sent to the framework until we get a response? (Didn't look but IIRC we hold onto them there)